recipe/meta.yaml

```diff
@@ -20,33 +20,37 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=39.0.0
+    - setuptools_scm >=3.4.3
     - wheel
   run:
+    # - importlib-metadata <4.3  # flake 8 requires an older version of importlib-metadata
     - autopep8 >=1.6.0,<1.7.0
     - flake8 >=4.0.0,<4.1.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.7.0
-    - pluggy
+    - mccabe >=0.6.0,<0.8.0  # flake 8 requires a version less than 0.7.0
+    - pluggy >=1.0.0
     - python-lsp-jsonrpc >=1.0.0
     - pycodestyle >=2.8.0,<2.9.0
     - pydocstyle >=2.0.0
     - pyflakes >=2.4.0,<2.5.0
     - pylint >=2.5.0
-    - python >=3.6
+    - python
     - rope >=0.10.5
     - setuptools >=39.0.0
     - ujson >=3.0.0
     - yapf
+    - whatthepatch
 
 test:
   imports:
     - pylsp
   requires:
     - pip
   commands:
-    - python -m pip check
     - pylsp --help
+    - python -m pip check
+    - python -m pip check
```

Comment:
Two lines are duplicates

Reply:
The duplicate line was removed. 
Also in order to solve the problem with `flake8`, the `pip check` section was commented out.  

https://github.com/AnacondaRecipes/python-lsp-server-feedstock/blob/69e81a93eb04fc78fc3f1ab0e9b2ada97f94ab33/recipe/meta.yaml#L52